app/utils/custom-fields.ts

```diff
@@ -168,8 +168,11 @@ export const buildCustomFieldValue = (
     }
 
     switch (def.type) {
-      case "BOOLEAN":
-        return { raw, valueBoolean: Boolean(raw) };
+      case "BOOLEAN": {
+        const finalValue =
+          typeof raw === "string" ? raw === "yes" : Boolean(raw);
+        return { raw: finalValue, valueBoolean: finalValue };
```

Comment:
I am not sure if this wont break something. Like maybe somewhere we check the raw value. I feel like in raw we should save the value from the sheet itself so "yes" or "no". What do you think?